### ISSUE TITLE
duplicate id warning from observeSequence

### ISSUE THREAD

**Issue by timtch** [meteor/meteor#1980]
**_1 Upvote**_ Warnings in a browser console with array:
duplicate id -#ffffff in ["#3c78aa", "#ffffff", "#ffffff", "#ffffff", "#ffffff"] observe_sequence.js:6

``` javascript
    var example = Pages.findOne().themes 

    JSON.stringify(example) 
      [ 
          { "colors": 
              [ "#3c78aa", "#ffffff", "#ffffff", "#ffffff", "#ffffff" ] 
          } 
      ]
```

**Comment by weeger** [comment]
+1

**Comment by hwillson** [comment]
The original reported issue has been addressed by PR #2227. Regarding the new issue reported in https://github.com/meteor/meteor/issues/1980#issuecomment-48137317, if someone is able to create and link to a small repro showing this, that would be great. Due to the fact the original issue has been addressed, and given the age of this issue, I'll close this for now. Happy to re-open if anyone feels this issue should still be addressed, and an issue reproduction can be provided. Thanks!

**Comment by smohantyCME** [comment]
Any status on removing the warnings?
